### ISSUE TITLE
Is `TwinkesCatalogZPoint` different from `PhoSimCatalogZPoint` ?`

### ISSUE THREAD

**Issue by rbiswas4** [DarkEnergyScienceCollaboration/Twinkles#326]
@jbkalmbach 

In  `Twinkles/python/desc/twinkles/generatePhosimInput.py` we use `Twinkles/python/desc/twinkles/twinklesCatalogDefs/TwinklesCatalogZPoint`. Is there a difference of this class with `PhoSimCatalogZPoint` in `sims_catUtils/python/lsst/sims/catUtils/exampleCatalogDefinitions/phoSimCatalogExamples.py`?

If this includes something more, should we be able to inherit most of the properties from `phoSimCatalogZPoint`?

**Comment by danielsf** [comment]
I know no one asked me, but it looks like `TwinklesCatalogZPoint` inherits a variability mixin designed to implement the AGN time delays.

Note: `TwinklesCatalogZPoint` will need to be fixed so that it writes 'rest frame' dust parameters before writing Milky Way dust parameters.

**Comment by rbiswas4** [comment]
> I know no one asked me  
:)  I just atted the username that was spitted out by blame.

> but it looks like TwinklesCatalogZPoint inherits a variability mixin designed to implement the AGN time delays.

But since it is adding properties (and not getting rid of anything) would it be possible to inherit all the `phosimZPoint` attributes or and add the time delay variability?

Following the usual advantages  of inheritance, when the catsim code got changed (eg. for the MW/host bug that you refer to), we would not have to come and change this code as well.
